packages/pf-codemods/scripts/buildReadme.ts

```diff
@@ -0,0 +1,42 @@
+import glob from "glob";
+import { join } from "path";
+import { readdir, readFile, writeFile } from "fs/promises";
+
+const rulePath = require
+  .resolve("@patternfly/eslint-plugin-pf-codemods")
+  .replace(/dist\/(esm|js)\/index\.js/, join("src", "rules", "v6"));
+
+const baseReadMePath = join(process.cwd(), "scripts", "baseReadMe.md");
+const baseReadMeContent = readFile(baseReadMePath, "utf-8");
+
+const v6RuleDirs = glob.sync(join(rulePath, "*"));
+
+const builtReadMes = v6RuleDirs.map(async (ruleDir) => {
+  const files = await readdir(ruleDir);
+
+  const readMe = files.find((file) => file.includes(".md"));
+  const input = files.find((file) => file.includes("Input.tsx"));
+  const output = files.find((file) => file.includes("Output.tsx"));
+
+  if (!readMe || !input || !output) {
+    console.error("Missing file in rule directory: ", ruleDir);
+    throw new Error("Missing file in rule directory");
```

Comment:
Do we need both the console and throw error, or could we just include the ruleDir in the thrown error message?

Reply:
Hmmmmm probably